The rest of the .py files use @PYTHON@ to pick up the exact path from
configure, but for some reason this one file does not.  This patch has
not yet been contributed upstream, but contribution is planned.

--- mailman-2.1.18-1/bin/msgfmt.py.orig	2014-05-06 09:43:56.000000000 -0700
+++ mailman-2.1.18-1/bin/msgfmt.py	2014-08-11 14:56:02.370461114 -0700
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! @PYTHON@
 # -*- coding: iso-8859-1 -*-
 # Written by Martin v. Löwis <loewis@informatik.hu-berlin.de>
 
